test/libgit2.jl

```diff
@@ -219,6 +219,13 @@ mktempdir() do dir
                 @test LibGit2.iszero(commit_oid2)
                 commit_oid2 = LibGit2.commit(repo, commit_msg2; author=test_sig, committer=test_sig)
                 @test !LibGit2.iszero(commit_oid2)
+                auths = LibGit2.authors(repo)
+                @test length(auths) == 3
```

Comment:
Pretty sure it's from scratch
On Jun 3, 2016 10:43 AM, "Tony Kelman" <notifications@github.com> wrote:

> In test/libgit2.jl
> <https://github.com/JuliaLang/julia/pull/16728#discussion_r65747279>:
>
> > @@ -219,6 +219,13 @@ mktempdir() do dir
> >                  @test LibGit2.iszero(commit_oid2)
> >                  commit_oid2 = LibGit2.commit(repo, commit_msg2; author=test_sig, committer=test_sig)
> >                  @test !LibGit2.iszero(commit_oid2)
> > +                auths = LibGit2.authors(repo)
> > +                @test length(auths) == 3
>
> which repo is this, is it constructed from scratch in this test harness,
> or is it referring to something like Example.jl that may change over time?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaLang/julia/pull/16728/files/4761fc6757a8ce5b5d8a049737ca7bc2a0293976#r65747279>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAyk452oBDr8VRy17iIXaMEOESjv-t9eks5qIGejgaJpZM4Is-63>
> .
>